lib/cluster/node.js

```diff
@@ -376,6 +376,15 @@ class ClusterNode {
           return;
         }
 
+        // Verify that no other node share the same IP address as this one
+        const duplicate = nodes.filter(node => node.ip === this.ip);
+
+        if (duplicate.length > 0) {
+          global.kuzzle.log.error(`[FATAL] Another node share the same IP address as this one (${this.ip}): ${duplicate[0].id}. Shutting down.`);
```

Comment:
This looks like @scottinet 's hotfix... Wasn't this living on a separate PR?

Reply:
IDK, I just based this branch on 2-dev, it seems pretty weird

Reply:
Ok, I thought you merged his branch while still in PR... BTW yes it's really weird git sees these lines as diff... Did you merge 2-dev on this branch lately? Even if it should make no difference, sinc diffs on GH are performed on the latest commit of the destination branch... OH! Wait! I see your destination branch is `master`!

Reply:
Ahh, you're right, I did put the wrong branch, I will changed it to 2-dev 👍

Reply:
Oh, right. It's fortunate you raised an issue about this one @xbill82: I saw this too, but just thought that it was some rebase mishap or something like that, and paid it no mind :sweat_smile: